src/jahia2wp.py

```diff
@@ -367,11 +368,17 @@ def export(site, wp_site_url, unit_name, to_wordpress=False, clean_wordpress=Fal
     else:
         theme_faculty = site.theme[default_language]
 
+    if not site.title[default_language]:
+        logging.warning("No wp tagline")
+        wp_tagline = "No-tagline"
```

Comment:
Sachant que dans `wp_site_title` tu mets la langue pour laquelle celui-ci n'est pas trouvé, ne serait-il pas pertinent de faire de même pour `wp_tagline` ?

Reply:
Ro l'autre, il commente ma PR qui est en WIP :-)

Reply:
Rho l'autre qui a pris soin de renommer sa PR avec [WIP] devant juste avant de me lâcher ce commentaire 😉

Reply:
Je ne comprends pas bien ton commentaire, il me semble que je fais la meme chose pour tagline que pour wp_site_title

Reply:
Je pense que Lucien proposait : `wp_tagline = "No-tagline-in-{}".format(default_language)` comme tu fais pour wp_site_title.

Reply:
Exact !